jscomp/others/js_re.mli

```diff
@@ -40,9 +40,25 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
 type t
+type result = string array
+
+val fromString : string -> t
```

Comment:
we need keep `external` instead of `val`, however, if it is  all about types, we only need `mli` file though